### PR TITLE
[Icons] Add Icon + Iconify tests

### PR DESCRIPTION
**Pull request by smnandre**
Add missing tests on Icon attributes (html + with/out)

(before making changes to the Renderer)

### CODE REVIEW

**Comment by norkunas** [comment]
How it's possible to use `attributes.without` when using `<twig:UX:Icon />` ?
I mean - I have css class that sets for all icons width/height, so there is no point to render icons with these attributes, how I could remove them from the output?

**Comment by smnandre** [comment]
This is planned, and i understand very well your need, as i have the same :)

**Comment by smnandre** [comment]
FYI this PR is one of the steps required ;)

File: src/Icons/src/Iconify.php [hunk 1]
```diff
@@ -26,12 +26,14 @@
  */
 final class Iconify
 {
+    private const API_BASE_URI = 'https://api.iconify.design';
```
Comment (kbond) [inline src/Icons/src/Iconify.php]: Can you use this in the configuration?

**Review by kbond — CHANGES_REQUESTED** [review]

File: src/Icons/src/Iconify.php (hunk 1 above)
Comment (smnandre) [reply src/Icons/src/Iconify.php:29]: Do we really need to make this configurable ? :| 

If "yes", i'd then let a null default value.

But using a const value in configuration ? That seems a bit weird. Not that i don't want, this just surprises me :)

Comment (smnandre) [reply src/Icons/src/Iconify.php]: It's done

**Comment by kbond** [comment]
Thank you Simon.

**Pull request merged**